### DISCUSSION TITLE
Converter to F20 Modification notice - Prior information notice 1370/07

### DISCUSSION THREAD

**Discussion by MarceauDebaille** [OP-TED/ted-xml-data-converter#30] [Q&A] [answered]
Hi,

the preview of  ted-xml-data-converter states : "All the standard TED XML forms using the R2.0.9 TED XML schema **(with the exceptions of F14 Corrigendum and F20 Modification**) are now convertible:"

Could you please inform me of the date when the converter (TED XML Data Converter) will be available to proceed with a rectification pre-information notice?

My client seeks to issue a rectification notice of prior information (2007/1370 regulation).
Regrettably, we have been advised by TED eforms that due to the constraints of the new e-forms regulation, effective since 2024, the publication of an amendment notice is presently unattainable. It appears that the current converter lacks the functionality required to support this endeavor.

Do you have any ideas for overcoming this difficulty?

kind regards,

**Comment by pdonohoe** [comment, marked as answer]
Dear @MarceauDebaille , that information is out-of-date. I will try to get it updated.
F20 notices can be converted, but only the original contract information will be included, the modifications will be excluded. This is because the modification section in TED F20 notices uses descriptive text, while eForms uses a more specific, granular format.
